src/ripple/ledger/impl/PaymentSandbox.cpp

```diff
@@ -180,6 +213,19 @@ PaymentSandbox::balanceHook (AccountID const& account,
     return adjustedAmt;
 }
 
+std::uint32_t
+PaymentSandbox::ownerCountHook (AccountID const& account,
+    std::uint32_t count) const
+{
+    std::uint32_t result = count;
+    for (auto curSB = this; curSB; curSB = curSB->ps_)
+    {
+        if (auto adj = curSB->tab_.ownerCount (account))
+            result = std::max (result, *adj);
```

Comment:
@scottschurr We never store the value that `ownerCountHook` computes into the ledger. It's only ever used when computing the reserve in the payment. This code is the only use of the value read from `ownerCountHook`: https://github.com/seelabs/rippled/blob/ebb0923accd3cde4fdc9d1de630ca0277b4fad37/src/ripple/ledger/impl/View.cpp#L141-L145